README.md

```diff
@@ -21,22 +20,193 @@ One line of code per option is all you need to write. For that, you get a nice
 automatically-generated help page, robust option parsing, and sensible defaults
 for everything you don't specify.
 
+This code is a feature-fork of Optimist: https://github.com/ManageIQ/optimist
+
+See the **Extended Features** section below for the differences/enhancements
+
 ## Features
 
-- Dirt-simple usage.
-- Single file. Throw it in lib/ if you don't want to make it a Rubygem dependency.
+- Simple usage.
```

Comment:
Optimist has always been single file, so I'd like to keep this aspect (I noticed the new chronic file)... we can discuss... maybe it is time to abandon that aspect

Reply:
@Fryguy 

This was intentional and used to solve a longstanding inconsistency with Optimist.  

Specifically, the behavior of Optimist changes depending on the existence of the chronic gem being installed, which is at best confusing and at worst going to really hose up people who don't use bundler. 

This is one of the few 'breaking' changes that I recall: please see https://github.com/nanobowers/optimist_xl/issues/4
for an explanation of how a user would explicitly enable use of chronic style dates.

I get that Optimist has always been single-file and dont object to merging it into one file, though that seems like much less of an issue now that it was prior to mass adoption of rubygems.

Reply:
the original author really emphasized this is simple and a single file.
At that time, copy and paste into `vendor` or `lib` was the best way to install code.

I vote splitting it up. That will make history checks more difficult but will make current day better.

---
@nanobowers I like chronic date - or something like `natural_date` that isn't chronic specific. (thought that is not backwards compatible for your scripts)

Reply:
@kbrock - i'm not opposed to Chronic, but the behavior of Optimist changes silently based on whether or not it is installed and IMO that's not good.  I would imagine (esp since this is a tool for handling cmdline options) that not everyone operates in the realm of running bundler - many companies or academic institutions have a site-wide ruby and gem install.  If IT-Dept installs chronic, scripts using Optmist start parsing DD-MM-YYYY as MM-DD-YYYY.  oops.

Likewise, the unittests behave differently if chronic is/isnt installed - though it is listed as a dev dependency and not a runtime dependency.  The change was intended to make things a lot more explicit - have the script call out that it wants to use a :chronic_date instead of a regular :date.

I moved the chronic handling into a separate file, but it could go back into the main file - separating felt appropriate for me since using explicitly require'ing it would require chronic which is not a strict runtime dependency.